src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule, Type } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TypeOfHelpComponent } from './components/type-of-help/type-of-help.component';
+import { MatIconModule } from '@angular/material/icon';
+import { MatCardModule } from '@angular/material/card';
+import { TranslateModule } from '@ngx-translate/core';
+
+const declarations: Type<any>[] = [TypeOfHelpComponent];
+
+const modules: Type<any>[] = [CommonModule, MatIconModule, MatCardModule, TranslateModule];
+
+@NgModule({
+  declarations: [...declarations],
+  imports: [...modules],
+  exports: [...modules, ...declarations],
+})
+export class SharedModule {}
```

Comment:
I'd opt for a module-per-component approach (like for e.g Angular Material does) instead of a shared module. 
For starters - It's gonna be next to impossible to find unused imports here after a few more components are added and removed.

Reply:
Sorry, forgot. You mentioned it earlier.